charts/timescaledb-single/docs/admin-guide.md

```diff
@@ -36,6 +36,8 @@ The following table lists the configurable parameters of the TimescaleDB Helm ch
 | `image.pullPolicy`                | The pull policy                             | `IfNotPresent`                                      |
 | `image.repository`                | The image to pull                           | `timescale/timescaledb-ha`                       |
 | `image.tag`                       | The version of the image to pull            | `pg13-ts2.1-latest`
+| `linkerd.enabled`                 | Enabled limited linkerd support.            | `false`
```

Comment:
seems we have two parameters to toggle a single action (shutdown checkpoint)?

Reply:
Fair enough, will refactor.

Reply:
Done.